helmfile/50-applications.yaml

```diff
@@ -149,52 +161,6 @@ releases:
   values:
   - values/cluster-admin-rbac.yaml.gotmpl
 
-# starboard-operator-psp-rbac
-- name: starboard-operator-psp-rbac
-  namespace: monitoring
-  labels:
-    app: starboard-operator-psp-rbac
-    group: starboard
-  chart: ./charts/starboard/starboard-operator-psp-rbac
-  version: 0.1.0
-  installed: {{ .Values.starboard.enabled }}
-
-# starboard-operator
-- name: starboard-operator
-  namespace: monitoring
-  labels:
-    app: starboard-operator
-    group: starboard
-  chart: ./upstream/starboard-operator
-  version: 0.9.1
-  installed: {{ .Values.starboard.enabled }}
-  values:
-  - values/starboard/starboard-operator.yaml.gotmpl
-
-# vulnerability-exporter (for starboard -> prometheus)
-- name: vulnerability-exporter
-  namespace: monitoring
-  labels:
-    app: vulnerability-exporter
-    group: starboard
-  chart: ./charts/starboard/vulnerability-exporter
-  version: 0.1.0
-  installed: {{ .Values.starboard.enabled }}
-  values:
-  - values/starboard/vulnerability-exporter.yaml.gotmpl
-
-# ciskubebench-exporter (for starboard -> prometheus)
-- name: ciskubebench-exporter
-  namespace: monitoring
-  labels:
-    app: ciskubebench-exporter
-    group: starboard
-  chart: ./charts/starboard/ciskubebench-exporter
-  version: 0.1.0
-  installed: {{ .Values.starboard.enabled }}
-  values:
-  - values/starboard/ciskubebench-exporter.yaml.gotmpl
-
```

Comment:
You need to add in the migration steps (I think there's an uninstall snippet already) to remove these releases.